src/bun.js/bindings/bindings.cpp

```diff
@@ -6568,3 +6568,40 @@ extern "C" double Bun__JSC__operationMathPow(double x, double y)
 {
     return operationMathPow(x, y);
 }
+
+CPP_DECL unsigned int Bun__CallFrame__getLineNumber(JSC::CallFrame* callFrame, JSC::JSGlobalObject* globalObject)
+{
+    auto& vm = JSC::getVM(globalObject);
+    JSC::LineColumn lineColumn;
+    String sourceURL;
+
+    JSC::StackVisitor::visit(callFrame, vm, [&](JSC::StackVisitor& visitor) -> WTF::IterationStatus {
+        if (Zig::isImplementationVisibilityPrivate(visitor))
+            return WTF::IterationStatus::Continue;
+
+        if (visitor->hasLineAndColumnInfo()) {
+            String currentSourceURL = Zig::sourceURL(visitor);
+
+            if (!currentSourceURL.startsWith("builtin://"_s) && !currentSourceURL.startsWith("node:"_s) && !currentSourceURL.endsWith("jest.zig"_s) && !currentSourceURL.contains("node_modules"_s) && currentSourceURL.length() > 0) {
```

Comment:
the `builint:` and the `node:` check can stay, but the rest of this should be deleted. it will never show .zig files. node_modules can be the source origin in a workspace in certain cases.